eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -210,22 +210,25 @@ public final Attr attr(final int pos) {
                 )
             );
         }
-        int idx;
-        for (idx = 0; idx < pos; ++idx) {
+        Attr attr = this.attr(this.order.get(0));
+        for (int idx = 0; idx <= pos; ++idx) {
             if (idx >= this.order.size()) {
                 throw new ExFailure(
                     String.format(
-                        "There are just %d attributes here, can't read the %d-th one",
-                        this.order.size(), pos
+                        "%s has just %d attribute(s), can't read the %d-th one",
+                        this,
+                        this.order.size(),
+                        pos
                     )
                 );
             }
             final String name = this.order.get(idx);
+            attr = this.attr(this.order.get(idx));
```

Comment:
@mximp We call it on every loop iteration. Apparently it's time consuming. What do you think?

Reply:
@volodya-lombrozo We have to get the attribute on each iteration to check wether it's vararg or not. Vararg makes the iteration to stop.